spec/models/user_spec.rb

```diff
@@ -3,96 +3,20 @@
 RSpec.describe(User, type: :model) do
   let(:user) { build(:user) }
 
-  context 'when name field is empty' do
-    it 'is invalid' do
-      user.name = ''
-      expect(user).to_not be_valid
-    end
+  context 'when user name is empty' do
```

Comment:
I would write all test cases under the single `describe 'validations'` block without contexts, they don't add much sense
@okohuch what do you think?

Reply:
@vikabelous I agree

Reply:
all shoulda's functions are self-explanatory.

Reply:
@shevchukm please do changes and share it with the teammates.

Reply:
@vikabelous , @okohuch    done